### PR TITLE
can run without a little lora config

### PR DESCRIPTION
**Pull request by aolemila**
This PR belongs to a part of LoRA. So this PR does have a related issue.

## Checklist

- [ ] Please use English, otherwise it will be closed.
- [ ] The purpose of the PR, or link existing issues this PR will resolve.
- [ ] The test plan, such as providing test command.
- [ ] (Optional) The necessary documentation update.

## Command

```bash
JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache JAX_USE_SHARDY_PARTITIONER=0 python3 -u -m sgl_jax.launch_server --model-path deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B --trust-remote-code  --tp-size=1 --device=tpu --mem-fraction-static=0.8 --chunked-prefill-size=2048 --download-dir=/tmp --dtype=bfloat16 --max-running-requests 2 --skip-server-warmup --page-size=64  --max-total-tokens=257536 --random-seed=27 --precompile-token-paddings=2048 --precompile-bs-paddings=2 --disable-overlap-schedule --enable-lora --max-lora-rank 1 --lora-target-modules gate_proj up_proj
```

image

### CODE REVIEW

File: python/sgl_jax/srt/managers/tp_worker.py [hunk 1]
```diff
@@ -454,6 +456,10 @@ def forward_batch_generation(
                 model_worker_batch
             )
 
+        # Prepare LoRA batch if LoRA is enabled
+        if self.server_args.enable_lora:
+            self.get_model_runner().lora_manager.prepare_lora_batch(forward_batch)
+
```
Comment (JamesBrianD) [inline python/sgl_jax/srt/managers/tp_worker.py]: Have call the `prepare_lora_batch` in scheduler run_batch

Comment (aolemila) [reply python/sgl_jax/srt/managers/tp_worker.py]: I have modified `prepare_lora_batch` according to discussion.
